Assets/Scripts/PlayerMoveScript.cs

```diff
@@ -0,0 +1,276 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+public class PlayerMoveScript : MonoBehaviour
+{
+    [SerializeField]
+    private Rigidbody2D Thisrigidbody;
+    [SerializeField]
+    private float AccelrationForce =5;
```

Comment:
Consider adding ToolTips to clarify floats:
https://docs.unity3d.com/ScriptReference/TooltipAttribute.html

Reply:
https://docs.google.com/document/d/1yrKCY_sVAdWkwgWAjJLiJFj1OOHn6dC__pGkSVB61YY/edit#heading=h.35gp8tzfruuq

Reply:
If it's obvious, like initializing the rigid body there, I can see no tool tip. But for values that the designer needs to tweak, tooltips are essential.

Reply:
Basically we should have very few if any comments and instead summaries and tool tips.